exec/java-exec/src/main/java/org/apache/drill/exec/physical/base/PhysicalOperatorUtil.java

```diff
@@ -46,10 +46,13 @@ private PhysicalOperatorUtil() {}
   }
 
   /**
-   * Helper method to create a list of MinorFragmentEndpoint instances from a given endpoint assignment list.
+   * Helper method to create a list of MinorFragmentEndpoint instances from a
+   * given endpoint assignment list.
    *
-   * @param endpoints Assigned endpoint list. Index of each endpoint in list indicates the MinorFragmentId of the
-   *                  fragment that is assigned to the endpoint.
+   * @param endpoints
```

Comment:
nit: no need to start from a new line.

Can you add Java docs to `@return`, please? :)